src/model.jl

```diff
@@ -490,48 +578,176 @@ const perihelion_parameters_phase = [251.9025, 280.8325, 128.3057, 348.1074, 292
                                      204.6609, 106.5938, 250.4676, 332.3345, 27.3039]
-
-
-
-# Eqn. 4 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
-function e_sincos_pi(year)
+  
+  
+  
+  # Eqn. 4 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+  function e_sincos_pi(year)
   M    = eccentricity_parameters_amplitude
   g    = eccentricity_parameters_mean_rate
   beta = eccentricity_parameters_phase
-
+  
   arg = deg2rad.(g/3600) * (year-1950) .+ deg2rad.(beta)
-
+  
   return sum(M .* sin.(arg)), sum(M .* cos.(arg))
-end
-
-eccentricity(year, e_sin_pi, e_cos_pi) = sqrt(e_sin_pi^2 + e_cos_pi^2)
-
-eccentricity(year) = eccentricity(year, e_sincos_pi(year)...)
-
-
-# Eqn. 1 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
-function obliquity(year)
+  end
+  
+  eccentricity(year, e_sin_pi, e_cos_pi) = sqrt(e_sin_pi^2 + e_cos_pi^2)
+  
+  eccentricity(year) = eccentricity(year, e_sincos_pi(year)...)
+  
+  
+  # Eqn. 1 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+  function obliquity(year)
   A     = obliquity_parameters_amplitude
   f     = obliquity_parameters_mean_rate
   delta = obliquity_parameters_phase
-
+  
   epsilon_star = 23.320556
-
+  
   return deg2rad(epsilon_star + sum(A/3600 .* cos.(deg2rad.(f/3600) * (year-1950) .+ deg2rad.(delta))))
-end
-
-
-# Eqn. 6 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
-function perihelion(year)
+  end
+  
+  
+  # Eqn. 6 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+  function perihelion(year)
   F     = perihelion_parameters_amplitude
   f     = perihelion_parameters_mean_rate
   delta = perihelion_parameters_phase
-
+  
   pi_ = atan(e_sincos_pi(year)...)
   psi_tilde = 50.439273
   zeta = 3.392506
-
+  
   psi = psi_tilde/3600 * (year-1950) + zeta + sum(F/3600 .* sin.(deg2rad.(f/3600) * (year-1950) .+ deg2rad.(delta)))
-
+  
   return (pi_ + deg2rad(psi)) % (2*pi)
+  end
+
+
+
+# possible years to pick annual sea ice extent for the northern hemisphere : 1979 to 2020
+# returns a vector with sea ice extent per month from March to February
+
+function read_sea_ice_extent_N(year=1979)
+
+  matrix = readdlm(joinpath(@__DIR__,"..","input/sea_ice_data/","N_ALL_extent_v3.0.dat"),comments=true)
+  yearindex = findfirst(isequal(year),matrix[:,1])
+  annual_sea_ice_extent = matrix[yearindex,2:13]
+
+  return annual_sea_ice_extent
 end
+
+
+
+"""
+calc_geography_per_month_extent()
+Determines a new geography matrix by determining sea ice extent with a monthly rate of change
+and mapping extent or reduction to adjacent sea ice cells and ocean cells on the Northern Hemisphere.
+
+"""
+
+function calc_geography_per_month_extent(geography,annual_sea_ice_extent, time_step, nlongitude=128, nlatitude =65)
```

Comment:
```suggestion
function calc_geography_per_month_extent!(geography,annual_sea_ice_extent, time_step, nlongitude=128, nlatitude =65)
```
The convention is to add an admiration sign when a function modifies an argument. In that case, you don't need to write `return geography`, but simply `return`.